integration/kubernetes/helm-chart/monitor/templates/prometheus/rbac.yaml

```diff
@@ -0,0 +1,60 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+{{ if .Values.prometheus.enabled -}}
```

Comment:
Do you need to create new roles/rbac for Alluxio monitor or just reuse the ones for Alluxio helm chart?

Reply:
Only found the rbac config in csi directory.